src/components/LabEditorGUI/SwitchNode.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import switchSVG from '../../assets/icons/switch.svg';
+import longSwitchSVG from '../../assets/icons/switch-48port.svg';
+import changeSelected from '../../redux/actions/changeGUI';
+import {useDispatch, useSelector} from 'react-redux';
+
+const SwitchNode = ({id, inputs, outputs}: any) => {
```

Comment:
Avoid any type, please Ctrl-f for any type and fix them. You can shoot me discord messages if you need help finding the types.